test/unit/js/S3PersistorTests.js

```diff
@@ -158,7 +158,7 @@ describe('S3PersistorTests', function() {
         'metrics-sharelatex': Metrics,
         crypto
       },
-      globals: { console }
+      globals: { console, Buffer }
```

Comment:
Out of curiosity, why do we need this in node 12? What happens if we don't add it?

Reply:
`Buffer` is not defined downstream then.

```
  1) S3PersistorTests
       sendStream
         when a hash is supploed
           "before each" hook for "should not calculate the md5 hash of the file":
     WriteError: upload to S3 failed: Buffer is not defined
      at Object.wrapError (app/js/PersistorHelper.js:100:12)
      at Object.sendStream (app/js/S3Persistor.js:126:27)
      at Context.<anonymous> (test/unit/js/S3PersistorTests.js:488:37)
      at processImmediate (internal/timers.js:439:21)
```